cs3/app/registry/v1beta1/registry_api.proto

```diff
@@ -82,8 +82,9 @@ message ListAppProvidersRequest {
   // OPTIONAL.
   // Opaque information.
   cs3.types.v1beta1.Opaque opaque = 1;
-  // TODO(labkode): maybe add some filter?
-
+  // OPTIONAL.
+  // Return all known app providers for a given mimetype (default to all).
+  string mime_type = 2;
```

Comment:
Maybe use filters? 

https://github.com/cs3org/cs3apis/blob/4b6237bcf90e4eb7d5d12782a2c7f2292ffb401e/cs3/sharing/collaboration/v1beta1/collaboration_api.proto#L145-L164

Reply:
Ah, that's what is meant by `Filters`, I'll have a look - and sure, the semantic is that: being able to query the app registry by mimetype